assemble/Zoltan_integration.F90

```diff
@@ -2130,6 +2122,15 @@ subroutine update_particle_list_element(detector_list_array)
           deallocate(ndets_being_sent)
           cycle
        end if
+       !Check if any are being sent
+       do i = 1, size(ndets_being_sent)
+          if (ndets_being_sent(i)>0) then
+             call mpi_bcast(detector_list%total_attributes, 3, getPINTEGER(), i-1, MPI_COMM_FEMTOOLS, ierr)
+             assert(ierr == MPI_SUCCESS)
+             total_attributes=sum(detector_list%total_attributes)
+             exit
```

Comment:
What happens if there are more than one processes that want to send?

Reply:
This whole loop is now redundant. It was previously checking if any processors were sending, and if they were it would set the value for total_attributes from the passed in attribute_size value from that processor.